### DISCUSSION TITLE
Support for Observable Plot tooltips

### DISCUSSION THREAD

**Discussion by LucHendriks** [quarto-dev/quarto-cli#6040] [Feature Requests]
### Description

Currently Quarto is not up-to-date with the latest two releases of Observable Plot ([0.6.7](https://github.com/observablehq/plot/releases/tag/v0.6.7) and [0.6.8](https://github.com/observablehq/plot/releases/tag/v0.6.8)). In version 0.6.7 support was added for the built-in tooltip functionality when making a plot using Observable Plot. I think this is a very useful addition and saves the user of having to do some [workaround](https://observablehq.com/@mkfreeman/plot-tooltip) to add this functionality. I was wondering if there are currently any plans of updating the Observable Plot version to 0.6.7 or higher?

I look forward to hearing if there are any plans and if so when this release would be available. Thanks in advance!

Luc

**Comment by mcanouil** [comment]
Dependencies are updated once per release.
So it's likely going to be updated for 1.5.
Note that the next version is 1.4.

**Comment by cderv** [comment]
Just a note about related issue:  https://github.com/quarto-dev/quarto-cli/issues/6031

We updated to 0.5.6 in April for next 1.4 version: https://github.com/quarto-dev/quarto-cli/pull/5320

**Reply by LucHendriks** [reply]
Thank you very much for the info, do you have a rough timeline on when the (pre-)release of 1.5 will be available?

In the meanwhile I will try the solution offered in the issue #6031.

**Reply by mcanouil** [reply]
1.5 should be worked on in 3 to 5 months from now.

**Reply by mcanouil** [reply]
I am closing this feature requests as there is nothing to be done beside waiting for the schedule dependencies update which happen at every release.
